lib/src/types/c2_full_results.dart

```diff
@@ -0,0 +1,43 @@
+// ignore_for_file: invalid_annotation_target
+
+part of 'index.dart';
+
+@freezed
+class C2FullResults with _$C2FullResults {
```

Comment:
Does this class provide something separately from the `C2Results` class? or is it largely just a copy to make changes less destructive/easy to undo/allow more rapid iteration?

Reply:
Does this have something to do with differences between the result data from the webhook vs a GET call?